### PR TITLE
feat: remove user context

### PR DESCRIPTION
**Pull request by niktverd**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -141,5 +141,8 @@
     "*.{json,yaml,yml,md}": [
       "prettier --write"
     ]
+  },
+  "publishConfig": {
```
Comment (NikitaCG) [inline package.json:145]: we dont need it

File: src/constructor/BlogConstructorProvider.tsx [hunk 2]
```diff
@@ -16,7 +15,6 @@ export interface BlogConstructorProviderProps {
     locale?: Locale;
     router?: RouterContextProps;
     theme?: ThemeValueType;
-    user?: UserContextProps;
```
Comment (NikitaCG) [inline src/constructor/BlogConstructorProvider.tsx:19]: remove information about it from contexts readme

**Review by NikitaCG — CHANGES_REQUESTED** [review]
fix comments

File: package.json (hunk 1 above)
Comment (niktverd) [reply package.json:145]: ok

**Pull request merged**